google-cloud-storage/samples/storage_set_autoclass.rb

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_set_autoclass]
+require "google/cloud/storage"
+
+##
+# This is a snippet for showcasing how to set the autoclass
+# configuration of a bucket.
+#
```

Comment:
Overall looks good, thanks for working on this! Could you help add a clarifying comment as below in this set autoclass sample?
```
# Note: Only patch requests that disable autoclass are currently supported.
# To enable autoclass, you must set it at bucket creation time.
```

Reply:
Have added the note.